src/utils/storybook.ts

```diff
@@ -14,8 +14,8 @@ const getFutureValues = () => {
 
   while (last.diff(first, 'days') >= 0) {
     for (let length = 0; length < 24; length++) {
-      const temp = 15 + Math.random() * 10
-      const rh = 40 + Math.random() * 15
+      const temp = 20 + Math.random() * 10
```

Comment:
Might want to stick some negative numbers in? You could use a sine wave to get something that matches reality closer. So like 
```
const temp = Math.sin(i)*20 + Math.random()*10
```

Reply:
Done